test/javascripts/acceptance/core-sidebar-test.js

```diff
@@ -56,6 +59,21 @@ acceptance("Discourse Chat - Core Sidebar", function (needs) {
     });
   });
 
+  needs.hooks.beforeEach(function () {
+    withPluginApi("1.3.0", (api) => {
+      api.addUsernameSelectorDecorator((username) => {
+        if (username === "hawk") {
+          return `<span class="on-holiday">${emojiUnescape(
+            ":desert_island:"
+          )}</span>`;
+        }
+      });
+    });
+  });
+  needs.hooks.afterEach(function () {
+    resetUsernameDecorators();
```

Comment:
I don't think we need to call this as this has been configured after each test; https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/tests/helpers/qunit-helpers.js#L168